### PR TITLE
Folder Added : Assignment 9

### PR DESCRIPTION
**Pull request by sourabh-kanwade**

### CODE REVIEW

**Review by somesh6797 — CHANGES_REQUESTED** [review]
Please Remove Comments

**Comment by sourabh-kanwade** [comment]
removed comments.

File: Sourabh_Kanwade/html/Assignment 9/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+    font-size: 12px;
+    border-left: 1px solid #e3e8ed;
+}
+.recruiter > a {
+    color: #01439f;
+}
+.covid-message-container {
+    background: #ffeb9c;
+    height: 40px;
+
```
Comment (sourabh513) [inline Sourabh_Kanwade/html/Assignment 9/css/styles.css]: remove extra line

File: Sourabh_Kanwade/html/Assignment 9/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+    border: none;
+    border-left: 1px solid #e3e8ed;
+    border-right: 1px solid #e3e8ed;
+}
+.open-search select {
+    height: 45px;
+    font-size: 14px;
+    color: #7d7d7d;
+    font-weight: 400;
+
```
Comment (sourabh513) [inline Sourabh_Kanwade/html/Assignment 9/css/styles.css]: remove extra line

File: Sourabh_Kanwade/html/Assignment 9/home.html [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+                    <a class="col-6" href="">Recruiter's Login</a>
+                    <a class="col-6" href=""><strong>Buy Online</strong> </a>
+                </div>
+                <div class="col-1"></div>
+            </div>
+            <div class="row center">
+                <div class="col-1"></div>
+                <div class="col-2">
+                    <img
+                        src="https://static.timesjobs.com/images_cand/tj_images/personlisation/tj-logo-new.png"
```
Comment (sourabh513) [inline Sourabh_Kanwade/html/Assignment 9/home.html]: remove extra spaces

File: Sourabh_Kanwade/html/Assignment 9/home.html [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+                    <a class="login-btn" href="">Register</a>
+                </div>
+                <div class="col-1"></div>
+            </div>
+        </div>
+        <div class="row">
+            <div class="open-search">
+                <div class="center">
+                    <div class="col-1"></div>
+                    <input class="col-2" type="text" placeholder="Enter Skills, Designation, etc" />
```
Comment (sourabh513) [inline Sourabh_Kanwade/html/Assignment 9/home.html]: remove / at the end of the line

File: Sourabh_Kanwade/html/Assignment 9/home.html [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+                    <input class="col-2" type="text" placeholder="Enter Location" />
```
Comment (sourabh513) [inline Sourabh_Kanwade/html/Assignment 9/home.html]: remove / at the end of the line

**Review by sourabh513 — APPROVED** [review]

**Review by somesh6797 — APPROVED** [review]

**Pull request merged**